File: Semana 1/Bucles y condicionales (cuaderno)/Condicionales C++/8. Mayor de 3 numeros.cpp
#include <iostream>
using namespace std;

int main () {
    int num1, num2, num3; 
    cout << "Ingrese el primer numero: " << endl;
    cin >> num1;

    cout << "Ingrese el segundo numero: " << endl;
    cin >> num2;

    cout << "Ingrese el tercer numero: " << endl;
    cin >> num3;

    if (num1 > num2 && num1 > num3) {
        cout << "El mayor es: " << num1 << endl;
    } else if (num1 < num2 && num2 > num3) {
        cout << "El mayor es: " << num2 << endl;
    } else {
        cout << "El mayor es: " << num3 << endl;
    }
    return 0;
}

File: Semana 1/Bucles y condicionales (cuaderno)/Condicionales Python/8. Mayor de 3 numeros.py
num1 = int(input("Ingrese el primer numero: "))
num2 = int(input("Ingrese el segundo numero: "))
num3 = int(input("Ingrese el tercer numero: "))

if num2 < num1 > num3:
    print ("El mayor es: ", num1)
elif num1 < num2 > num3:
    print ("El mayor es: ", num2)
else:
    print ("El mayor es: ", num3)